Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.9-slim-bullseye
```

Comment:
You should follow this [Dockerfile](https://github.com/tdex-network/tdex-daemon/blob/master/Dockerfile).. we basically use 2 images, one for building all the binaries with all the required dependencies, another one where we install only the compiled binaries and set the CMD/ENTRYPOINT.